ayab-software.md

```diff
@@ -4,26 +4,18 @@ title: AYAB Software
 permalink: /ayab-software/
 ---
 
-All software components can be found in [our Git Repository](https://github.com/AllYarnsAreBeautiful/ayab-desktop).
+## Latest Version: v1.0
 
+[AYAB Manual for v1.0](https://manual.ayab-knitting.com/1.0/installation/software/)
 
-## Installation instructions for v0.95
+## Software Components, Source Code, Test Releases
 
-### Windows
+All software components can be found in [our Git Repositories](https://github.com/AllYarnsAreBeautiful).
 
-<p><a href="https://github.com/AllYarnsAreBeautiful/ayab-desktop/releases/download/v0.95/AYAB-Win10-v0.95-Setup.exe" target="\_blank"><button type="button" class="btn btn-lg btn-primary">AYAB 0.95 (Windows 11, 10 and 7)</button></a></p>
+Test Releases can be found in the [Releases Section](https://github.com/AllYarnsAreBeautiful/ayab-desktop/releases) of the AYAB-Desktop Repository.
```

Comment:
I'm not sure why we'd need to talk about Test Releases here? All releases can be found on https://github.com/AllYarnsAreBeautiful/ayab-desktop/releases.

Also I kind of liked the big download buttons? It certainly is more convenient for us to just link to another page that already has the information (e.g. https://github.com/AllYarnsAreBeautiful/ayab-desktop/releases/latest) but the GitHub release page can be intimidating for less technically-minded users.

Reply:
We can leave the test releases out.
We can use big buttons and link directly to the separate files (win/Mac/Linux) on the release page

Reply:
On the release page, you mean on GitHub? That could work.